### PR TITLE
FAT-6226

### PR DESCRIPTION
**Pull request by nayimovag**
[FAT-6226](https://issues.folio.org/browse/FAT-6226)
[C380562](https://foliotest.testrail.io/index.php?/cases/view/380562)
[jenkins run](https://jenkins-aws.indexdata.com/job/Testing/job/Cypress_tests/2988/)

### CODE REVIEW

File: cypress/support/fragments/bulk-edit/bulk-edit-search-pane.js [hunk 1]
```diff
@@ -738,6 +738,10 @@ export default {
   clickActionsRunBy(runByUsername) {
     cy.do(ListRow({ text: including(runByUsername) }).find(Button({ icon: 'ellipsis' })).click());
   },
+  
+  verifyNoActionsButton(runByUsername) {
+    cy.expect(ListRow({ text: including(runByUsername) }).find(Button({ icon: 'ellipsis' })).absent());
```
Comment (IhorBohdan) [inline cypress/support/fragments/bulk-edit/bulk-edit-search-pane.js:743]: Move Button to const

File: cypress/e2e/bulk-edit/logs/csv/bulk-edit-logs-csv-users-view-permission.cy.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  });
+
+  after('delete test data', () => {
+    FileManager.deleteFile(`cypress/fixtures/${userUUIDsFileName}`);
+    FileManager.deleteFile(`cypress/fixtures/${editedFileName}`);
+    Users.deleteViaApi(user.userId);
+    FileManager.deleteFileFromDownloadsByMask(`*${matchedRecordsFileName}`, changedRecordsFileName);
+  });
+
+  it('C380562 Verify generated Logs files for Users CSV are hidden without "Users: Can view user profile" permission (firebird)', { tags: [testTypes.smoke, devTeams.firebird] }, () => {
```
Comment (IhorBohdan) [inline cypress/e2e/bulk-edit/logs/csv/bulk-edit-logs-csv-users-view-permission.cy.js:42]: Does test type is smoke?

File: cypress/support/fragments/bulk-edit/bulk-edit-search-pane.js (hunk 1 above)
Comment (nayimovag) [reply cypress/support/fragments/bulk-edit/bulk-edit-search-pane.js]: done

File: cypress/e2e/bulk-edit/logs/csv/bulk-edit-logs-csv-users-view-permission.cy.js (hunk 2 above)
Comment (nayimovag) [reply cypress/e2e/bulk-edit/logs/csv/bulk-edit-logs-csv-users-view-permission.cy.js]: thanks, fixed

**Review by IhorBohdan — APPROVED** [review]

**Pull request merged**